app/assets/stylesheets/content/_choice.sass

```diff
@@ -26,10 +26,29 @@
 // See doc/COPYRIGHT.rdoc for more details.
 //++
 
+//a little hack, the width will be just fine for this element
+$choice-parent-relation-select-width: 325px
```

Comment:
I see... button is usually in the second row on smaller screens...
Can you include (very few/basic) details in the comment (i.e. that it is intended for IE 11)...

Some day we might not support IE 11 anymore and someone will be glad to ditch that hack ^^